packages/playground/client/src/index.ts

```diff
@@ -95,17 +95,11 @@ export async function startPlaygroundWeb({
 		progressbar: !disableProgressBar,
 	});
 	progressTracker.setCaption('Preparing WordPress');
+	// Set a default blueprint if none is provided.
 	if (!blueprint) {
-		const playground = await doStartPlaygroundWeb(
-			iframe,
-			setQueryParams(remoteUrl, {
-				['php-extension']: 'kitchen-sink',
-				'site-slug': siteSlug,
-			}),
-			progressTracker
-		);
-		onClientConnected(playground);
-		return playground;
+		blueprint = {
+			phpExtensionBundles: ['kitchen-sink'],
+		};
```

Comment:
Unrelated to the issue.

Playground failed to load when running it like this:
```
const client = await startPlaygroundWeb({
            iframe: document.getElementById('wp'),
            remoteUrl: `http://127.0.0.1:5400/remote.html`,
});
```

It required an empty blueprint to work. 

Looking more at the code it turns out that this condition if true wouldn't run some start actions like `collectPhpLogs`.

Reply:
Good spot, thank you for fixing this!